app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -117,12 +122,12 @@ const ContentListView: React.FC = () => {
       try {
         if (!isLoading) {
           setIsLoading(true);
-          const data = await findContent(
-            makeFilter({
-              ...filter,
-            }),
-          );
-          const page = new Page(data.page - 1, data.quantity, data?.items, data.total);
+          const result = await findContentWithElasticsearch(getFilter(filter, actions), false);
+          const items = result.hits?.hits?.map((h) =>
+            castContentToSearchResult(h._source as IContentModel),
+          ) as IContentSearchResult[];
```

Comment:
Why do you need to cast to `IContentSearchResult[]`?  It already should be that.

Reply:
Good catch, and it should have been cleaned up now, thanks